rpp-charter.md

```diff
@@ -3,46 +3,50 @@
-to be known collectively as RESTful Provisioning Protocol (RPP).
+to be known collectively as the RESTful Provisioning Protocol (RPP).
 These specifications will specifically target a REST architecture high on the Richardson Maturity Model
 using HTTPS and JSON. 
 
-The working group will evaluate for possible inclusion of extensibility considerations, allowing
-future use cases beyond the domain name provisioning as well as adding
-or extending operations and data models defined in the core set of protocol specifications.
+In addition to use cases and scenarios for domain name registration, this working group will
+consider features and extension mechanisms useful for other types of Internet registration areas
+(e.g. IP addresses and autonomous system numbers). 
 
 Industry experience of the ccTLDs with non-EPP provisioning protocols and use cases from EPP may influence the outputs
 of this working group, but direct compatibility of RPP with EPP is not a goal.
 
-The working group will consider functional equivalents of used functionality 
-of core EPP for domain names, hosts and contacts (RFC5730,5731,5732,5733)
-and possibilities of mapping data objects, operations, command and response.
-The working group will also consider functional equivalents of commonly
-used registered EPP extensions, either by providing RPP extensibility
-points or incorporating them into core of the protocol.
-New functionalities, not having any equivalents in EPP, may be also defined for RPP.
+This working group will consider functional equivalents of functionality 
+from EPP for domain names, hosts and contacts ([RFC5730](https://datatracker.ietf.org/doc/html/rfc5730),
+[5731](https://datatracker.ietf.org/doc/html/rfc5731), [5732](https://datatracker.ietf.org/doc/html/rfc5732), and
+[5733](https://datatracker.ietf.org/doc/html/rfc5733)) and mappings for data objects, operations, commands and response.
+This working group will also consider functional equivalents of registered EPP extensions, either through 
+similar RPP extensions or incorporating them into the core of the protocol.
+New functionalities, not having any equivalents in EPP, may be defined for RPP.
 
 The RPP working group is focused on designing a new protocol intended to co-exist alongside EPP, 
 supporting diverse needs in the ecosystem. As such, considerations for replacing EPP or migration 
-scenarios are outside the scope of this work.
+scenarios away from EPP are outside the scope of this work.
 
 The REGEXT working group is chartered to maintain and standardize extensions to EPP. Consequently, 
 any extensions or changes to EPP, including those related to RPP functionality that do not exist in EPP, 
-are explicitly out of scope for the RPP working group.
+are explicitly out of scope for the RPP working group. Some IETF participants active in the REGEXT
+working group will be involved in RPP working groups, and coordination between the two
```

Comment:
"will be involved in RPP working groups" -> "will be involved in the RPP working group"